mobika2_tests/launch/test4_static_obstacle.launch

```diff
@@ -0,0 +1,139 @@
+<launch>   
```

Comment:
Test 4 is a test of the navigation stack.    
Purpose:
* Test pose repeatability
* Test time to goal
Static obstacle not relevant

Reply:
Remove all static obstacle parts